persistence/src/main/java/edu/softserve/zoo/persistence/specification/impl/animal/AnimalGetAllByHouseIdSpecification.java

```diff
@@ -0,0 +1,30 @@
+package edu.softserve.zoo.persistence.specification.impl.animal;
+
+import edu.softserve.zoo.model.Animal;
+import edu.softserve.zoo.model.House;
+import edu.softserve.zoo.persistence.specification.hibernate.DetachedCriteriaSpecification;
+import org.hibernate.criterion.DetachedCriteria;
+import org.hibernate.criterion.Restrictions;
+
+/**
+ * Animal specification for retrieving animals in specific {@link House}
+ *
+ * @author Serhii Alekseichenko
+ */
+public class AnimalGetAllByHouseIdSpecification implements DetachedCriteriaSpecification<Animal> {
+    private static final String HOUSE_ID = House.class.getSimpleName().toLowerCase() + ".id";
```

Comment:
if this functionality needed around the project, please implement common method for this stuff